modules/lib/src/main/resources/assets/styles/wizard/content-wizard-panel.less

```diff
@@ -138,10 +138,10 @@
 
     .edit-permissions-button {
       order: 3;
-      padding: 15px;
+      //padding: 15px;
       cursor: pointer;
       z-index: 1;
-      line-height: @toolbar-height;
+      //line-height: @toolbar-height;
```

Comment:
It is always better to not leave commented stuff that pollutes code, just delete it if it is not needed